type-definitions/tests/immutable-flow.js

```
@@ -212,6 +201,14 @@ stringToNumber = Map({a: 1}).merge([[1, 'a']])
 // $ExpectError
 const stringToNumber: Map<string, number> = Map({a: 1}).merge(numberToString)
 
+/**
+ * FIXME: this should throw an error, the signature should be
+ *
+ * ```
+ * merger: (previous: V, next: V, key: K) => V,
+ *    ...iterables: Iterable<K, V>[]
+ * ```
+ */
 stringToNumber = Map({'a': 1}).mergeWith((previous, next, key) => 1, [1])
```

I don't understand why it should throw an error :(.